processing/src/main/java/io/druid/query/PrioritizedExecutorService.java

```diff
@@ -39,10 +41,11 @@
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicLong;
 
 public class PrioritizedExecutorService extends AbstractExecutorService implements ListeningExecutorService
 {
-  public static PrioritizedExecutorService create(Lifecycle lifecycle, ExecutorServiceConfig config)
+  public static PrioritizedExecutorService create(Lifecycle lifecycle, DruidProcessingConfig config)
```

Comment:
Because ExecutorServiceConfig is in java-util, so has further implications beyond just druid. And since this implementation is just in Druid and NOT part of java-util (yet) I didn't want to pollute java-util with properties that it does not support.